dash/dash.py

```diff
@@ -1332,7 +1331,8 @@ def enable_dev_tools(
 
         if dev_tools.silence_routes_logging:
             logging.getLogger("werkzeug").setLevel(logging.ERROR)
-            self.logger.setLevel(logging.INFO)
+
+        self.logger.setLevel(logging.INFO)
```

Comment:
We don't do all that much with `self.logger` but what we do we always want the user to see - not just when silencing built-in route logging.